OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	public class TurretedInfo : ITraitInfo, UsesInit<TurretFacingInit>, Requires<BodyOrientationInfo>, IActorPreviewInitInfo
+	public class TurretedInfo : ITraitInfo, UsesInit<TurretFacingInit>, Requires<BodyOrientationInfo>, Requires<AttackTurretedInfo>, IActorPreviewInitInfo
```

Comment:
I really don't like the idea of making this depend on `AttackTurreted`.  Can we do this a different way?